x-pack/plugins/observability_solution/dataset_quality/public/state_machines/dataset_quality_details_controller/state_machine.ts

```diff
@@ -524,11 +524,15 @@ export const createDatasetQualityDetailsControllerStateMachine = ({
       loadDegradedFields: (context) => {
         const { startDate: start, endDate: end } = getDateISORange(context.timeRange);
 
-        return dataStreamDetailsClient.getDataStreamDegradedFields({
-          dataStream: context.dataStream,
-          start,
-          end,
-        });
+        if (!context.isNonAggregatable) {
```

Comment:
My main concern with this approach is:
if they are being called in a parallel mode, then is this flag useful? we wouldn't call this only on refresh but the first time the call will take place. Am I missing something?

Reply:
This is the PR we discussed offline, i forgot to close it.
This fix is not valid 👍🏼